README.md

```diff
@@ -25,31 +25,31 @@ Overview
 * **Freedom to build what you want**  FreeCAD is an open-source parametric 3D 
 modeler made primarily to design real-life objects of any size. 
 Parametric modeling allows you to easily modify your design by going back into 
-your model history and changing its parameters. 
+your model history to change its parameters. 
 
-* **Create 3D from 2D & back** FreeCAD allows you to sketch geometry constrained
+* **Create 3D from 2D & back** FreeCAD lets you to sketch geometry constrained
  2D shapes and use them as a base to build other objects. 
  It contains many components to adjust dimensions or extract design details from 
- 3D models to create high quality production ready drawings.
+ 3D models to create high quality production-ready drawings.
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
  including product design, mechanical engineering and architecture. 
- Whether you are a hobbyist, a programmer, an experienced CAD user, 
- a student or a teacher, you will feel right at home with FreeCAD.
+ Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
+ you will feel right at home.
```

Comment:
Thanks for the edit. LGTM. But there is something nagging at me. 

> you will feel right at home

Frankly, this is presumptuous on our part. Maybe something like 'one can find benefit' or something similar. Make it more neutral